frontend/src/app/containers/ExperimentPage/ExperimentTourDialog.js

```diff
@@ -33,159 +32,89 @@ export default class ExperimentTourDialog extends React.Component {
-                  </div>
-                </div>
-              </div>
-              {step.copy &&
-                <div className="tour-text">
-                  <Localized id={this.l10nId(['tour_steps', idx, 'copy'])}>
-                    <p>{step.copy}</p>
-                  </Localized>
-                </div>}
-            </div>
-          ))}
-          <div className="tour-actions">
-            <div onClick={() => this.tourBack()}
-                 className={classnames('tour-back', { hidden: atStart })}><div/></div>
-            <div onClick={() => this.tourNext()}
-                 className={classnames('tour-next', { 'no-display': atEnd })}><div/></div>
-            <Localized id="tourDoneButton">
-              <div onClick={e => this.complete(e)}
-                   className={classnames('tour-done', { 'no-display': !atEnd })}>Done</div>
-            </Localized>
-          </div>
-        </div>
-      </div>
-    );
+    return enabled ? (
+        <Localized id="tourOnboardingTitle" $title={experiment.title}>
+          <h3 className="modal-header">{experiment.title} enabled!</h3>
+        </Localized>) : (<h3 className="modal-header">{experiment.title}</h3>);
   }
 
-  renderDots(tourSteps: Array<any>, currentStep: number) {
-    const dots = tourSteps.map((el, index) => {
-      if (currentStep === index) return (<div key={index} className="current dot"></div>);
-      return (<div key={index} className="dot" onClick={e => this.tourToDot(e, index)} ></div>);
-    });
-
-    return dots;
-  }
-
-  cancel(e: Object) {
-    e.preventDefault();
-    this.props.sendToGA('event', {
-      eventCategory: 'ExperimentDetailsPage Interactions',
-      eventAction: 'button click',
-      eventLabel: 'cancel tour'
-    });
-    if (this.props.onCancel) { this.props.onCancel(e); }
+  renderStep(tourSteps: Array<Object>, currentStep: number) {
+    return tourSteps.map((step, idx) => (idx === currentStep) && (
+        <div key={idx} className="step-content">
+          <div className="step-image">
+            <img src={step.image} />
+          </div>
+          {step.copy &&
+            <div className="step-text">
+              <Localized id={this.l10nId(['tour_steps', idx, 'copy'])}>
+                <p>{step.copy}</p>
+              </Localized>
+            </div>}
+        </div>
+    ));
   }
 
-  complete(e: Object) {
-    e.preventDefault();
-    this.props.sendToGA('event', {
-      eventCategory: 'ExperimentDetailsPage Interactions',
-      eventAction: 'button click',
-      eventLabel: 'complete tour'
-    });
-    if (this.props.onComplete) { this.props.onComplete(e); }
-  }
+  render() {
+    const { sendToGA, experiment } = this.props;
+    const { currentStep } = this.state;
+    const tourSteps = experiment.tour_steps || [];
 
-  tourToDot(e: Object, index: number) {
-    e.preventDefault();
-    this.setState({ currentStep: index });
+    const myProps = {
```

Comment:
Same as the other comment - would be better to have this stuff defined once on the class, rather than on every render()